### PR TITLE
Rename to raw_code, compiled_code

### LINKED ISSUE #292: [CT-937] Rename `raw_sql` -> `raw_code`, `compiled_sql` -> `compiled_code`
See: https://github.com/dbt-labs/dbt-core/pull/5499

### PR DESCRIPTION
**Pull request by jtcohen6**
resolves #292

Needed for compatibility with https://github.com/dbt-labs/dbt-core/pull/5499 in v1.3.0-b1

### CODE REVIEW

File: src/app/docs/model.js [hunk 1]
```diff
@@ -36,8 +36,8 @@ angular
         $scope.parentsLength = Object.keys($scope.parents).length;
         var default_compiled = '\n-- compiled SQL not found for this model\n';
```
Comment (emmyoop) [inline src/app/docs/model.js:37]: We should update all these comments too.

Comment (jtcohen6) [reply src/app/docs/model.js:37]: yes we should :)

Comment (jtcohen6) [reply src/app/docs/model.js:37]: ...and syntax highlighting, potentially

Comment (ChenyuLInx) [reply src/app/docs/model.js]: Updated this but couldn't find where to touch to update syntax highlighting

**Review by jtcohen6 — COMMENTED** [review]
This is enough to unblock the beta (= functioning `dbt-docs` site). I reflected the next round of improvements in #294, #295. Could one of you approve + merge, so we can distribute to `dbt-core` ahead of cutting the beta?

**Comment by jtcohen6** [comment]
Update: managed to figure out syntax highlighting over in #295, with a little help from a real one

File: src/app/components/code_block/code_block.js [hunk 2]
```diff
@@ -5,28 +5,40 @@ const $ = require('jquery');
 
 const css = require("./code_block.css")
 
+function getLanguageClass(language) {
+    if (language == 'python') {
+        return 'language-python';
+    } else {
+        return 'language-sql';
+    }
```
Comment (stu-k) [inline src/app/components/code_block/code_block.js:13]: Nitpick:
```suggestion
    return language === 'python'
        ? 'language-python'
        : 'language-sql';
```

File: src/app/docs/model.js [hunk 3]
```diff
@@ -30,14 +30,18 @@ angular
...
+        // how to dynamically assign ng-class in code_block.html based on this scope variable?
+        // $scope.language = "language-" + (mod.language || "sql");
         $scope.references = dag_utils.getReferences(project, mod);
         $scope.referencesLength = Object.keys($scope.references).length;
         $scope.parents = dag_utils.getParents(project, mod);
         $scope.parentsLength = Object.keys($scope.parents).length;
-        var default_compiled = '\n-- compiled SQL not found for this model\n';
+        $scope.language = mod.language;
+
+        var default_compiled = '\n-- compiled code not found for this model\n';
```
Comment (stu-k) [inline src/app/docs/model.js:41]: Nitpick:
```suggestion
        const default_compiled = '\n-- compiled code not found for this model\n';
```

**Review by stu-k — APPROVED** [review]
Only non-blocking nitpicks, nothing else stands out 👍

**Pull request merged**